src/Ghcid.hs

```diff
@@ -52,6 +53,7 @@ options = cmdArgsMode $ Options
     ,arguments = [] &= args &= typ "MODULE"
     ,test = Nothing &= name "T" &= typ "EXPR" &= help "Command to run after successful loading"
     ,warnings = False &= name "W" &= help "Allow tests to run even with warnings"
+    ,nostatus = False &= name "S" &= help "Suppress status messages"
```

Comment:
I suspect you also want `& name "no-status"` here, or to rename the field `no_status`, so that it gets the flag `--no-status` rather than `--nostatus`.